views/index.jade

```diff
@@ -14,7 +15,7 @@ block content
           input#email.form-control(type='email', placeholder='Enter Email', name='email', required='')
         .form-group
           label.control-label(for='git_url') GITURL:
-          input#git_url.form-control(type='text', placeholder='Enter URL of Github Repository', name='git_url')
+          select#git_url.form-control(type='text', placeholder='Enter URL of Github Repository', name='git_url')
```

Comment:
OK I saw what you did here -> https://github.com/sch00lb0y/yaydoc/blob/0b0b930791cb20be51d2171fa521d738911ad67d/public/scripts/suggestion.js#L11